hurumap/templates/profile/profile_detail.html

```diff
@@ -962,6 +963,8 @@ <h1>Livestock</h1>
         var chartRelease = thisRelease ? thisRelease : dataRelease;
         var chartReleaseYear = chartData['metadata']['year'];
 
+        var colorbrewer = {{ WAZIMAP.theme.color_schemes | safe }};
```

Comment:
Lets use `HURUMAP....` since these are HURUmap specific settings (I know `WAZIMAP` will also works but it's just the principle of it :sunglasses:)

Reply:
@kilemensi the context processor is configure in wazimap, thus the `WAZIMAP.theme.color_schemes`

Reply:
Shame...